src/drt/src/dr/FlexDR_maze.cpp

```diff
@@ -1849,10 +1851,10 @@ void FlexDRWorker::routeNet_prep(
 {
   frBox3D* tbx = nullptr;
   if (getDRIter() >= beginDebugIter)
-    cout << "Creating dest search points from pins:\n"; 
+    logger_->info(DRT, 2004, "Creating dest search points from pins:\n"); 
   for (auto& pin : net->getPins()) {
     if (getDRIter() >= beginDebugIter)
-      cout << "Pin " << pin->getName() << "\n";
+      logger_->info(DRT, 2005, "Pin " + pin->getName() + "\n");
```

Comment:
remove the \n as the logger handles that.  better would be

logger_->info(DRT, 2005, "Pin {}", pin->getName());